cpp/path_module/path_module.cpp

```diff
@@ -6,6 +6,15 @@ extern "C" int mgp_init_module(struct mgp_module *module, struct mgp_memory *mem
   try {
     mgp::memory = memory;
 
+    auto empty_list = mgp::Value(mgp::List{});
+    auto empty_map = mgp::Map{};
+    empty_map.Insert("key", empty_list);
+
+    AddProcedure(Path::Create, Path::kProcedureCreate, mgp::ProcedureType::Read,
+                 {mgp::Parameter(Path::kCreateArg1, mgp::Type::Node),
+                  mgp::Parameter(Path::kCreateArg2, {mgp::Type::Map, mgp::Type::List}, mgp::Value(empty_map))},
+                 {mgp::Return(Path::kResultCreate, mgp::Type::Path)}, module, memory);
```

Comment:
I suggest changing the second argument/`relationships` to a list type:
* in the algorithm, you only work with `relationships["rel"]`
* simpler calls, e.g. `path.create(club, {rel: [inLeague, inCountry, inConfederation]})`→ `path.create(club, [inLeague, inCountry, inConfederation])`
* consistent with APOC

Reply:
The problem is that null value can't be sent in a list ex. [relationship, relationship, null] but it can be sent this way. This is important because the main usage of this function is with the optional clause

Reply:
I see, thanks!